Pipfile

```diff
@@ -22,9 +22,9 @@ httmock = "*"
 "flake8-mock" = "*"
 "flake8-print" = "*"
 "flake8-tuple" = "*"
-moto = "*"
-google-compute-engine = "*"
+moto = "==1.3.4"
```

Comment:
This had to be done because moto doesn't support the latest boto, so they pinned to an old version. We can't run on that version so had to pin boto as well.